File: webapp-php-symfony/src/AppBundle/Entity/Calendar.php
<?php
namespace AppBundle\Entity;

use Doctrine\Common\Collections\ArrayCollection;
use Doctrine\ORM\Mapping as ORM;

/**
 * @ORM\Entity
 */
class Calendar
{
    /**
     * @ORM\Id
     * @ORM\Column(type="bigint")
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;

    /**
     * @ORM\OneToMany(targetEntity="AppBundle\Entity\Appointment", mappedBy="calendar")
     */
    private $appointments;

    /**
     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\Person", inversedBy="calendars")
     * @ORM\JoinColumn(name="person_id", referencedColumnName="id", nullable=false)
     */
    private $owner;

    /**
     * Constructor
     */
    public function __construct()
    {
        $this->appointments = new ArrayCollection();
    }

    /**
     * Get id
     *
     * @return integer
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Add appointment
     *
     * @param Appointment $appointment
     * @return Calendar
     */
    public function addAppointment(Appointment $appointment)
    {
        $this->appointments[] = $appointment;

        return $this;
    }

    /**
     * Remove appointment
     *
     * @param Appointment $appointment
     */
    public function removeAppointment(Appointment $appointment)
    {
        $this->appointments->removeElement($appointment);
    }

    /**
     * Get appointments
     *
     * @return \Doctrine\Common\Collections\Collection
     */
    public function getAppointments()
    {
        return $this->appointments;
    }

    /**
     * Set owner
     *
     * @param Person $owner
     * @return Calendar
     */
    public function setOwner(Person $owner = null)
    {
        $this->owner = $owner;

        return $this;
    }

    /**
     * Get owner
     *
     * @return Person
     */
    public function getOwner()
    {
        return $this->owner;
    }
}

File: webapp-java-spring/src/main/java/de/ehmke/entity/Calendar.java
package de.ehmke.entity;

import javax.persistence.*;
import java.util.Set;

@Entity
@Table(name = "calendar")
public class Calendar
{

    /**
     * The id
     */
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    private Long id;

    /**
     * The person
     */
    @ManyToOne()
    @JoinColumn(nullable = false, name = "person_id")
    private Person person;

    /**
     * The appointments
     */
    @OneToMany(mappedBy = "calendar", cascade = CascadeType.REMOVE)
    private Set<Appointment> appointments;

    /**
     * @return the id
     */
    public Long getId()
    {
        return this.id;
    }

    /**
     * @param id the id
     */
    public void setId(Long id)
    {
        this.id = id;
    }

    /**
     * @return the person
     */
    public Person getPerson()
    {
        return person;
    }

    /**
     * @param person the person
     */
    public void setPerson(Person person)
    {
        this.person = person;
    }

    /**
     * @return the appointments
     */
    public Set<Appointment> getAppointments()
    {
        return appointments;
    }

    /**
     * @param appointments the appointments
     */
    public void setAppointments(Set<Appointment> appointments)
    {
        this.appointments = appointments;
    }
}
